MonoGame.Framework/Windows/WinFormsGamePlatform.cs

```diff
@@ -145,14 +170,39 @@ public override bool BeforeUpdate(GameTime gameTime)
         public override bool BeforeDraw(GameTime gameTime)
         {
             return true;
-        }
-
-        public override void EnterFullScreen()
-        {
+        }
+
+
+        public override void EnterFullScreen()
+        {
+#if (WINDOWS && DIRECTX)
+            // FIXME : FULLSCREEN
+
+            if (_window._form.Location.X != 0 && _window._form.Location.Y != 0)
+            {
+                _savedWindowedLocX = _window._form.Location.X;
+                _savedWindowedLocY = _window._form.Location.Y;
+            }
+            _window.IsBorderless = true;

```

Comment:
Shouldn't we just be using `_window._form.WindowState = FormWindowState.Maximized`?